templates/model/templates/affiliated_committer_list.html

```diff
@@ -0,0 +1,18 @@
+{% extends "model/templates/base.html" %}
+{% block title %}
+<title>Affiliated Contributors List</title>
+{% endblock %}
+{% block content %}
+<h1>Affiliated Contributors List</h1>
+{% if affiliated_committer_list %}
+<ul>
+  {% for affiliatedcommitter in affiliated_committer_list %}
```

Comment:
affiliatedcommitter -> affiliated_committer

(same applies everwhere you are not using snake_case. jinja is based on python ; same naming conventions apply.